common/lib/capa/capa/templates/codeinput.html

```diff
@@ -1,12 +1,16 @@
+<%page expression_filter="h"/>
 <%!
 from django.utils.translation import ugettext as _
 from openedx.core.djangolib.markup import HTML
 %>
-<section id="textbox_${id}" class="capa_inputtype textbox cminput">
-  <textarea rows="${rows}" cols="${cols}" name="input_${id}" 
-            aria-label="${_("{programming_language} editor").format(programming_language=mode)}" 
-            aria-describedby="answer_${id}" 
-            id="input_${id}" 
+<div id="textbox_${id}" class="capa_inputtype textbox cminput">
```

Comment:
@cptvitamin don't we need role=group here since we changed it from section to div.